ext/standard/var.c

```diff
@@ -585,6 +587,54 @@ PHP_FUNCTION(var_export)
 }
 /* }}} */
 
+#define RETURN_VAR_TYPE(key) \
+	RETURN_INTERNED_STR(CG(known_strings)[key]);
+
+/* {{{ proto string var_type(mixed var)
+   Get the type of a variable. */
+PHP_FUNCTION(var_type)
+{
+	zval *var;
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "z", &var) == FAILURE) {
+		return;
+	}
+
+	switch (Z_TYPE_P(var)) {
+		case IS_ARRAY:
+			RETURN_VAR_TYPE(ZEND_STR_ARRAY);
+
+		case IS_FALSE:
+		case IS_TRUE:
+			RETURN_VAR_TYPE(ZEND_STR_BOOL);
+
+		/* case IS_FLOAT: */
```

Comment:
Has this been left in for documentation purposes? Or is this a case that should be added back in?